build/Microsoft.DotNet.Cli.Test.targets

```diff
@@ -70,6 +70,8 @@
       <TestDirectory>$(RepoRoot)/test/</TestDirectory>
       <DotnetUnderTest>$(Stage2Directory)</DotnetUnderTest>
     </PropertyGroup>
+
+    <MakeDir Directories="$(TestPackagesDir)" Condition="!Exists('$(TestPackagesDir)')"/>
```

Comment:
We need this. One of the problems that I ran into is that some test packages depend on CLi.Utils and InternalAbstractions and we want to use the ones from the repo, so that we can test that code. Before this change, what was happening was that these packages were being restored to the ones in the feed and not the local ones. It worked by chance. Now that this is the first rc3 build, there is no package in the feed for it to just work. So we need this.